src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace System.Net
 {
-    internal enum CookieVariant
+    public enum CookieVariant
```

Comment:
Serialization of Cookie/CookieContainer is/was broken between different frameworks and I was looking into it. So our tests in fact failed correctly and you were just faster finding it